wallet/createtx_test.go

```diff
@@ -26,7 +25,7 @@ func TestTxToOutputsDryRun(t *testing.T) {
 	defer cleanup()
 
 	// Create an address we can use to send some coins to.
-	addr, err := w.CurrentAddress(0, waddrmgr.KeyScopeBIP0044)
+	addr, err := w.CurrentAddress(0, waddrmgr.KeyScopeBIP0084)
```

Comment:
Why the change in key-scope? To force a segwit addr?

Reply:
No, I don't think I actually wanted to commit this change. Fixed, thanks!